src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -143,6 +143,11 @@ public void start(Map<String, String> props, ElasticsearchClient client) {
 
       writer = builder.build();
       writer.start();
+      log.info(
```

Comment:
"we know this value from the configs - is this superfluous" -lev

Reply:
If you have the code, then it is superfluous. But if you don't, it can be very confusing to know which config controls this value in the log. I've seen cases when users were trying to control things like this, but they were definitely unsure which property set it. It shouldn't take trial and error to get that to work.